presto-docs/src/main/sphinx/release/release-0.275.rst

```diff
@@ -0,0 +1,30 @@
+=============
+Release 0.275
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix Disaggregated Coordinator to correctly identify running queries as leaked.
+* Add :func:`trimmed_mean` for :ref:`tdigest <tdigest_type>`.
+* Add session property ``query_max_output_positions`` and configuration property ``query.max-output-positions`` to control how many rows a query can output. The query might end up returning more rows than the limit as the check is asynchronous.
+* Add :func:`construct_tdigest()` for `Tdigest`.
```

Comment:
nit: maybe keep it in the previous line so functions are together.

Reply:
Should we use :ref:`tdigest <tdigest_type>`, same as above?

Added a new commit instead of ammending, so I can cherry-pick back to master branch